react-events/my-app/src/CustomButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+
+interface CustomButtonProps {
+  text: string;
+  color: string;
+  onCustomClick: (text: string) => void;
+}
+
+const CustomButton: FC<CustomButtonProps> = ({
+  text,
+  color,
+  onCustomClick,
+}) => {
+  const buttonStyle: React.CSSProperties = {
+    backgroundColor: color,
+    color: 'white',
+    padding: '10px',
+    margin: '5px',
+    border: 'none',
+    borderRadius: '4px',
+    cursor: 'pointer',
+  };
+
+  const handleClick = () => {
```

Comment:
This function is no longer needed. It was only used in the first part of the exercise and then moved to the parent component.